front/app/hooks/useTextingCampaign.ts

```diff
@@ -5,7 +5,7 @@ import {
 } from 'services/textingCampaigns';
 import { isNilOrError } from 'utils/helperUtils';
 
-export default function useTextingCampaigns(campaignId) {
+export default function useTextingCampaign(campaignId: string) {
```

Comment:
Added types. Really wish we could turn on TS Config strict.

Reply:
I'm not super familiar with TS config but it might be possible to use on a per-file basis: https://blog.allegro.tech/2021/09/How-to-turn-on-TypeScript-strict-mode-in-specific-files.html

That may be too complicated and too much overhead vs. the return it would bring but maybe we can discuss as a team